<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogClass</class>
 <widget class="QDialog" name="DialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Peers</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QListWidget" name="peerList"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="refreshButton">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Activate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="filesButton">
     <property name="text">
      <string>Files</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="updateNumOfPeers">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>peerList</tabstop>
  <tabstop>refreshButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
